src/index.html

```diff
@@ -13,6 +13,58 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="container"
+      data-qa="card"
+    >
+      <div class="container__image">
+        <img
+          class="container__image-img"
+          src="../src/images/imac.jpeg"
+          alt="mac"
+        />
+      </div>
+
+      <div class="container__product">
+        <div class="container__product-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+      </div>
+
+      <div class="container__code">Product code: 195434</div>
+
+      <div class="container__stars">
+        <div class="container__stars-list">
```

Comment:
Fix the classes and add the modifier classes
```suggestion
        <div class="stars stars--4">
```